src/ray/object_manager/object_directory.h

```diff
@@ -104,6 +104,11 @@ class ObjectDirectoryInterface {
   virtual ray::Status ReportObjectRemoved(const ObjectID &object_id,
                                           const ClientID &client_id) = 0;
 
+  /// Get Local client id
```

Comment:
(nit) Please update the documentation to the following: "Get local client id."

Reply:
Done.